### PR TITLE
[Feature-1] new features were implemented

### PR DESCRIPTION
**Pull request by eZipcoder**
The following changes were made:

- new file just added
- home page added
- the title of the homepage changed

### CODE REVIEW

File: feat.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+new file just added
```
Comment (Erfan-Motallebi) [inline feat.txt:1]: This feature was awesome

Comment (eZipcoder) [reply feat.txt:1]: thank you for the consideration

**Review by Erfan-Motallebi — APPROVED** [review]
Good Job ezipcoder

**Review by Erfan-Motallebi — APPROVED** [review]

File: feat/home.htm [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Erfan-Motallebi) [inline feat/home.htm:6]: can be better

**Review by Erfan-Motallebi — CHANGES_REQUESTED** [review]
please change the title of the page to homepage

**Review by Erfan-Motallebi — COMMENTED** [review]
good decision

**Pull request closed without merging**